recipe/conda_forge_ci_setup/upload_or_check_non_existence.py

```diff
@@ -89,6 +100,16 @@ def distribution_exists_on_channel(binstar_cli, meta, fname, owner, channel='mai
 
 
 def upload_or_check(recipe_dir, owner, channel, variant):
+    # If there is a dedicated upload in the recipe use that one
+    if 'upload_or_check_non_existence.py' in os.listdir(recipe_dir):
+        import importlib.util
+        spec = importlib.util.spec_from_file_location('upload_two',
+                                                      os.path.join(recipe_dir,
```

Comment:
```suggestion
                                                      os.path.join(recipe_dir, "conda_forge_ci_setup",
```

Reply:
Ah so the plan is to include the entire ci setup folder?